spec/codelog/command/bump_spec.rb

```diff
@@ -62,15 +62,15 @@
 
       it 'prints an error message' do
         expect_any_instance_of(described_class).to receive(:abort).with Codelog::Message::Error.invalid_version_type('bla')
-        subject.run 'bla', '12-12-2012'
+        subject.run 'bla', '12-12-2012', preview: false
       end
     end
   end
 
   describe '.run' do
     it 'creates an instance of the class to run the command' do
-      expect_any_instance_of(described_class).to receive(:run).with 'major', '12-12-2012'
-      described_class.run 'major', '12-12-2012'
+      expect_any_instance_of(described_class).to receive(:run).with 'major', '12-12-2012', preview: false
+      described_class.run 'major', '12-12-2012', preview: false
```

Comment:
Is it not possible to create a test with the `preview` set as `true`, like you did in the test file `release_spec.rb`?

Reply:
@rmohashi Added on 88614924cef8506d377fe5ab079d5a944f1468b2.

Reply:
:+1: